app/DiscordMessages/Add.php

```diff
@@ -23,7 +23,7 @@ public function message(): string
         
         $this->getContractInfo($parts[1]);
 
-        if (isset($parts[3])) {
+        if (isset($parts[3])) { // do we have more than one coop name parameter?
```

Comment:
Not my best lines of code here. May come back and rewrite this differently sometime.